spec.html

```diff
@@ -26137,21 +26390,21 @@ <h1>Cyclic Module Records</h1>
                 [[RequestedModules]]
               </td>
               <td>
-                a List of Strings
+                a List of ModuleRequest Records
               </td>
               <td>
-                A List of all the |ModuleSpecifier| strings used by the module represented by this record to request the importation of a module. The List is in source text occurrence order.
+                A List of all the |ModuleSpecifier| strings with the relative import attributes used by the module represented by this record to request the importation of a module. The List is in source text occurrence order.
```

Comment:
The use of "relative" seems possibly wrong here? "related", or just leave out entirely?

This whole sentence seems awkward to read IMO

Reply:
How about this?

```suggestion
                A List of the ModuleRequest Records associated with the imports in this module. The List is in source text occurrence order of the imports.
```